agent/src/beerocks/monitor/monitor_rssi.h

```diff
@@ -43,6 +43,8 @@ class monitor_rssi {
     void send_rssi_measurement_response(std::string &sta_mac, monitor_sta_node *sta_node);
     void monitor_idle_station(std::string &sta_mac, monitor_sta_node *sta_node);
 
+    bool arp_enabled() { return arp_socket > 0; }
```

Comment:
Here also add the initiative_arp config check.

Reply:
No the inactive arp is only for inactive arp, and not related to general arp.